docs/src/content/how-to/polyfills.raw.md

```diff
@@ -0,0 +1,59 @@
+## Polyfills
+
+We're using [polyfill.io](https://polyfill.io) ([Financial-Times/polyfill-service](https://polyfill.io/)) for polyfills:
+
+> Polyfill.io is a service which makes web development less frustrating by selectively polyfilling just what the browser needs. Polyfill.io reads the User-Agent header of each request and returns polyfills that are suitable for the requesting browser.
+
+For static sites we're using the hosted version by Polyfill.io.  
+For server side rendered sites we're using the local version of polyfill.io.
```

Comment:
Can you add reasoning? Why would we add this 'complexity'?

Note that I am not doubting we need to, but it's good to state it in the documentation.

Reply:
mark